core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
```

Comment:
Similar to https://github.com/novoda/bintray-release/blob/feature/support_for_gradle_plugins/core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy#L45 this method is about handling (several) defaults (here with project as parameter). An early exit would not allow to handle more defaults in the future.

One could have extracted the setting of versionAttributes in a method, but that seemed overkill here.